### ISSUE TITLE
Badge sometimes doesn't get populated on browser start

### ISSUE THREAD

**Issue by mikeyaworski** [mikeyaworski/Twitch-Lurker#28]
The extension will sit with no badge sometimes

**Comment by oKtosiTe** [comment]
Is this the reason for needing to click the extension to get it going? Twitch Lurker won't do anything for me until I click it after browser startup. It's been that way for months.

**Comment by mikeyaworski** [comment]
Yes, that's the issue. It happens very rarely for me. I didn't think it would be happening frequently to others, but if it is, then I'll look into it soon.

**Comment by oKtosiTe** [comment]
I can't say for sure how consistently it happens for me, but it's been consistent enough that I assumed it was a new security measure imposed on extensions by Firefox. If there _is_ anything that can be done about it, that would be great!

**Comment by oKtosiTe** [comment]
I should probably also note that I use FrankerFaceZ pretty heavily, in case that might interfere with Twitch Lurker. I need FFZ for accessibility reasons, but if you'd like me to test with it disabled I can certainly do that.

**Comment by mikeyaworski** [comment]
Ah, you're using Firefox. That might be why it happens more frequently for you. I'll test using that browser to see if I can reproduce it more consistently.

And no need to disable any other extensions - they won't interfere with Twitch Lurker.
